analytics/terraform/spark-k8s-operator/addons.tf

```diff
@@ -336,11 +399,34 @@ module "eks_data_addons" {
 
   #---------------------------------------------------------------
   # Spark Operator Add-on
+  # Add this to enable YuniKorn as Default Scheduler
+  #    controller:
+  #      batchScheduler:
+  #        enable: true
+  #        default: "yunikorn"
   #---------------------------------------------------------------
   enable_spark_operator = true
   spark_operator_helm_config = {
-    version = "1.4.2"
-    values  = [templatefile("${path.module}/helm-values/spark-operator-values.yaml", {})]
+    version = "2.0.2"
+    values = [
+      <<-EOT
+        spark:
+          # -- List of namespaces where to run spark jobs.
+          # If empty string is included, all namespaces will be allowed.
+          # Make sure the namespaces have already existed.
+          jobNamespaces:
+            - default
+            - spark-team-a
+            - spark-team-b
+            - spark-team-c
+          serviceAccount:
+            # -- Specifies whether to create a service account for the controller.
+            create: false
```

Comment:
I don't see that we're creating a service account outside of the helm chart for the spark-operator (yet). 
We may need to create a custom Service Account for Pot Template access, but shouldn't we have the helm chart create the Service Account for RBAC permissions and such?